test/e2e/framework/pv_util.go

```diff
@@ -92,124 +91,181 @@ type PersistentVolumeClaimConfig struct {
 }
 
 // Clean up a pv and pvc in a single pv/pvc test case.
-func PVPVCCleanup(c clientset.Interface, ns string, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) {
-	DeletePersistentVolumeClaim(c, pvc.Name, ns)
-	DeletePersistentVolume(c, pv.Name)
+// Note: delete errors are appended to []error so that we can attempt to delete both the pvc and pv.
+func PVPVCCleanup(c clientset.Interface, ns string, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) []error {
+
+	var errs []error
+	var err error
+
+	if pvc != nil {
+		if err = DeletePersistentVolumeClaim(c, pvc.Name, ns); err != nil {
+			errs = append(errs, fmt.Errorf("Failed to delete PVC %q. Error: %v", pvc.Name, err))
```

Comment:
capital case looks wrong if you append it to another error, as in: `fmt.Fprintf("some error text up the call stack: %v", yourErr)`. In pkg/ we have 681 with capital-case, 2436 with lower-case. In golang's standard library it's 25 against 721, so much much clearer.